internal/core/src/clustering/KmeansClustering.h

```diff
@@ -61,8 +61,11 @@ class KmeansClustering {
     GetRemoteCentroidsObjectPrefix() const {
         auto index_meta_ = file_manager_->GetIndexMeta();
         auto field_meta_ = file_manager_->GetFieldDataMeta();
-        return file_manager_->GetChunkManager()->GetRootPath() + "/" +
-               std::string(ANALYZE_ROOT_PATH) + "/" +
+        std::string path = file_manager_->GetChunkManager()->GetRootPath();
```

Comment:
how about using boost::filesystem::path and just 
return path /path1 /path2/ ...;